src/firestore/firekit.ts

```diff
@@ -1165,7 +1167,7 @@ export class RoarFirekit {
     this._verifyAuthentication();
     this._verifyAdmin();
 
-    if ([name, dateOpen, dateClose, assessments].some((param) => param === undefined || param === null)) {
+    if ([name, publicName,dateOpen, dateClose, assessments].some((param) => param === undefined || param === null)) {
```

Comment:
I think if we fallback on the `name` param, then we don't want to force `publicName` to be required.
```suggestion
    if ([name, dateOpen, dateClose, assessments].some((param) => param === undefined || param === null)) {
```